run.py

```diff
@@ -46,8 +49,10 @@ def main(gear_context):
 if __name__ == "__main__":
 
     with flywheel_gear_toolkit.GearToolkitContext() as gear_context:
+        api_key = gear_context.get_input("api_key")["key"]
```

Comment:
This should be placed into the parse_config.generate_gear_args() function and then passed as gear args for the necessary steps.

Reply:
All fw should be moved removed from run.py and placed into parse_config.py where all args are parsed from config and passed as needed back to run.py.